lib/galaxy/tools/actions/upload_common.py

```diff
@@ -283,7 +284,7 @@ def _chown( path ):
         try:
             # get username from email/username
             pwent = trans.user.system_user_pwent(trans.app.config.real_system_username)
```

Comment:
This is already done in the system_user_pwent function by the following code:
```Python
# Checks if real_system_username is either user_email or username
       else:
           log.warning("invalid configuration of real_system_username")
           system_user_pwent = None
```
A log warning should be issued when the variable is not defined.